hellabot.go

```diff
@@ -35,6 +35,8 @@ type Bot struct {
 	// Log15 loggger
 	log.Logger
 	didJoinChannels sync.Once
+	// For thread-safe access to triggers slice
+	triggersMu *sync.Mutex
```

Comment:
put this next to the `triggers` field.

Reply:
Yep. Tested both ways and it makes sense to use a plain object here.